notebooks/query-expansion.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.            expanded_query = eval(result['llm']['replies'][0]) + [query]
</pre><p>maybe not a blocker, but using  <code>eval</code> is considered an unsafe practice.</p><p>Would <code>json.loads</code> work? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>